recipes/pyrosm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.5.2" %}
+{% set name = "pyrosm" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cf16e00a4523b4d73f440dc38f15db1f1889e5a5173596fbd09173a9fc4e5089
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython >=0.28
+    - pyrobuf >=0.9.3
+    - cykhash
+    - python-rapidjson
+    - geopandas
+    - shapely
+    - pygeos
```

Comment:
Do you really need all of these at build time?

Reply:
Probably not. Still learning, so don't have yet a good understanding which packages I should include in the `build` and which ones to `run`.  I guess at least cython related packages (`Cython` + `pyrobuf`) should be listed here in the build but maybe I could try to remove the others.

Reply:
@scopatz Hmm, well after a couple of try removing things from host produces errors. All those packages mentioned in `host` are hard dependencies for this library.. Hence, I guess I should include all of them? Sorry these might be silly questions, but the [conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host) are not very clearly describing what you actually should put here..

Reply:
Hmm, well thinking about this and going through the codes makes me quite certain, yes I do need all of those libraries during build time, as those libraries are used from cython files (`.pyx`).

Reply:
OK Just making sure!